### PR TITLE
workflows: switch to org-wide golangci-lint job

### PR DESCRIPTION
**Pull request by roman-khimov**

### CODE REVIEW

File: .golangci.yml [hunk 1]
```diff
@@ -1,73 +0,0 @@
-# This file contains all available configuration options
-# with their default values.
```
Comment (roman-khimov) [inline .golangci.yml:2]: One thing that bugs me is that this can affect local linter runs.

Comment (roman-khimov) [reply .golangci.yml:2]: A make target to download it from .github can be added.

**Review by AnnaShaleva — APPROVED** [review]

File: .golangci.yml (hunk 1 above)
Comment (AnnaShaleva) [reply .golangci.yml:2]: It should be updated manually if any changes are included into .github's config. Other than that it works OK on my machine.

Comment (roman-khimov) [reply .golangci.yml:2]: Yeah, the only other way to sync is submodules, but it's _so_ inconvenient (especially the need to bump them).

Comment (roman-khimov) [reply .golangci.yml:2]: I think the main thing is that we're still guarded by GH linter job, so if local config is out of sync we'll still see linter job failure and it should be sufficient.

Comment (AnnaShaleva) [reply .golangci.yml:2]: Let’s keep the current way, it’s good.

**Review by tatiana-nspcc — APPROVED** [review]

**Pull request merged**